### PR TITLE
feat - added modal

### PR DESCRIPTION
**Pull request by ashishnnnnn**
Added Modal with cancel button and text in it ..

### CODE REVIEW

**Comment by SanjayJatti** [comment]
Your component is looking amazing. Code is all clean. Great job man.

**Review by ishapatell1 — APPROVED** [review]
Modal looks good to go. Merge it with the dev branch.

**Review by VedanthB — APPROVED** [review]
looks good to go 🚀

File: modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  width: 55%;
+  text-align: center;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  background-color: #bb453073;
```
Comment (ZuberDunge) [inline modal/modal.css]: use color variables for colors, it's a good practice and very useful

**Review by ZuberDunge — COMMENTED** [review]
looks good to me

File: modal/modal.css (hunk 1 above)
Comment (ashishnnnnn) [reply modal/modal.css]: Done

File: modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  position: relative;
+  min-height: 18rem;
+  word-break: break-all;
+  border-radius: 10px;
+}
+
```
Comment (lannisterrr) [inline modal/modal.css:23]: looks good to me.

**Pull request merged**